cmd/access/node_builder/access_node_builder.go

```diff
@@ -747,6 +754,9 @@ func (builder *FlowAccessNodeBuilder) extraFlags() {
 				return errors.New("state-stream-response-limit must be greater than or equal to 0")
 			}
 		}
+		if builder.executionDataCollCatchupEnabled && !builder.executionDataSyncEnabled {
```

Comment:
if execution data sync is enabled, why not always index from execution data instead of only trigger when behind?

Reply:
this is just a quick fix for master. We'll be adding a proper integration in the next few months